tools/accuracy_checker/accuracy_checker/annotation_converters/antispoofing.py

```diff
@@ -0,0 +1,110 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import os
+from .format_converter import DirectoryBasedAnnotationConverter, ConverterReturn
+from ..representation import ClassificationAnnotation, ContainerAnnotation
+from ..utils import read_json, check_file_existence
+from ..config import PathField
+
+class AntispoofingDatasetConverter(DirectoryBasedAnnotationConverter):
+    __provider__ = 'antispoofing'
+    annotation_types = (ClassificationAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        configuration_parameters = super().parameters()
+        configuration_parameters.update(
+            {
+                'data_dir': PathField(
+                    is_directory=True, optional=False,
+                    description='path to input images'
+                ),
+                'meta_file': PathField(
+                    description='path to json file with dataset meta (e.g. label_map)', optional=True
+                ),
+                'annotations': PathField(
+                    description='path to json file with dataset annotations ({index : {path: ..., labels: ..., boxes: ... (optional)}})', optional=False
+                )
+            }
+        )
+        return configuration_parameters
+
+    def configure(self):
+        super().configure()
+        self.data_dir = self.get_value_from_config('data_dir')
+        self.annotations = self.get_value_from_config('annotations')
+        self.meta = self.get_value_from_config('meta_file')
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        """Reads data from disk and returns dataset in converted for AC format
+
+        Args:
+            check_content (bool, optional): Check if content is valid. Defaults to False.
+            progress_callback (bool, optional): Display progress. Defaults to None.
+            progress_interval (int, optional): Units to display progress. Defaults to 100 (percent).
+
+        Returns:
+            [type]: Converted dataset
+        """
+        annotation_tuple = self.generate_annotations()
+        annotations = []
+        content_errors = None if not check_content else []
+        meta = self.generate_meta()
+        labels_to_id = meta['label_map']
+        num_iterations = len(annotations)
+
+        for i, (img_name, label, bbox) in enumerate(annotation_tuple):
+            image_annotation = ClassificationAnnotation(img_name, label)
+            if bbox:
+                image_annotation.metadata['rect'] = bbox
+            annotations.append(image_annotation)
+
+            if check_content:
+                if not check_file_existence(os.path.join(self.data_dir, img_name)):
```

Comment:
we prefer to use pathlib.Path objects for working with path. please replace.
More over, data_dir is already Path object, so you can simplify
```suggestion
                if not check_file_existence(self.data_dir /img_name):
```